src/explore-education-statistics-common/src/components/AccordionSection.tsx

```diff
@@ -103,7 +104,13 @@ const AccordionSection = ({
       </div>
       <div
         aria-labelledby={headingId}
-        className={classNames(classes.sectionContent)}
+        className={classNames(
+          classes.sectionContent,
+          styles.sectionContent,
```

Comment:
Think `sectionContent` could just be applied on mount. I would expect `sectionContent` to essentially mirror what `govuk-accordion__section-content` does i.e. hide the content, but using `visibility: hidden` for the lazy loading.

As it doesn't get applied until mount, the non-JS version's content should be visible still.